packages/imperative/src/config/src/api/ConfigSecure.ts

```diff
@@ -242,6 +242,17 @@ export class ConfigSecure extends ConfigApi {
         return secureProps;
     }
 
+    /**
+     * Retrieve secure properties for a given layer path
+     *
+     * @param layerPath Path of the layer to get secure properties for
+     * @returns the secure properties for the given layer, or null if not found
+     */
+    public secureFieldsForLayer(layerPath: string): IConfigSecureProperties {
```

Comment:
Is adding this method necessary? I'm probably being dumb but it seems like the `secureFields` method has a layer parameter which can be used to filter the secure fields returned for only a specific layer.